server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -0,0 +1,56 @@
+import itertools
+from planning.feed_parsers.onclusive import OnclusiveFeedParser
+from typing import List
+from superdesk import get_resource_service
+
+
+def _get_cv_items(_id: str) -> List:
+    cv = get_resource_service("vocabularies").find_one(req=None, _id=_id)
+    return cv["items"]
+
+
+class CPOnclusiveFeedParser(OnclusiveFeedParser):
+    """
+    Superdesk -CP event parser
+
+    Feed Parser which can parse the Onclusive API Events
+    """
+
+    event = []
+
+    def parse(self, content, provider=None):
+        onclusive_cv_items = _get_cv_items("onclusive_ingest_categories")
+        anpa_categories = _get_cv_items("categories")
+        items = super().parse(content, provider)
+
+        for item in items:
+            category = []
+            if item.get("subject"):
+                for subject in item.get("subject"):
```

Comment:
I think we might be getting also other subject codes there so would be good to check the subject `scheme` if these are categories and not any other metadata